### ISSUE TITLE
MenuItem aria-label just repeats the visible text

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#12501]
Noticed while looking at https://github.com/WordPress/gutenberg/pull/12309

The `MenuItem` component has an `aria-label` attribute that just repeats the visible text of the menu item. For example:
```
<button type="button" aria-label="Visual Editor" aria-checked="true" role="menuitemradio" class="...">
    Visual Editor
</button>
```

While it doesn't harm anything, it's redundant. Not sure if there's a specific reason for the aria-label but, unless I'm missing something, maybe it could be simply removed?

**Comment by timwright12** [comment]
I can knock this one out today (via @10up)
